experiments/RunAnalysisBO.py

```diff
@@ -0,0 +1,85 @@
+import matplotlib
```

Comment:
Could you create an entire folder for the BOLearner experiment and place all his files there?
Also add the robot yaml file instead of modifing the one from the example.